ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -102,9 +109,14 @@ if [[ -s "${SHARED_DIR}/xpn.json" ]]; then
   HOST_PROJECT_COMPUTE_SERVICE_ACCOUNT="$(jq -r '.computeServiceAccount' "${SHARED_DIR}/xpn.json")"
   HOST_PROJECT_CONTROL_SERVICE_ACCOUNT="$(jq -r '.controlServiceAccount' "${SHARED_DIR}/xpn.json")"
   HOST_PROJECT_PRIVATE_ZONE_NAME="$(jq -r '.privateZoneName' "${SHARED_DIR}/xpn.json")"
-else
-  # Set HOST_PROJECT to the cluster project so commands with `--project` work in both scenarios.
-  HOST_PROJECT="${PROJECT_NAME}"
```

Comment:
Seem like the new logic changed the behavior, in the original logic, when it is not a `xpn` install, the gcloud option would be `--project="${PROJECT_NAME}"`, but the new logic will change that to `--project=''`

Reply:
If the specified project (by "--project") is the current one, either with "--project" or not would be the same. 
```
$ gcloud config get project
openshift-qe
$ 
$ gcloud --project=openshift-qe dns record-sets list --zone cam-src-28960-pzq7z-private-zone
NAME                                                    TYPE  TTL    DATA
cam-src-28960.qe.gcp.devcluster.openshift.com.          NS    21600  ns-gcp-private.googledomains.com.
cam-src-28960.qe.gcp.devcluster.openshift.com.          SOA   21600  ns-gcp-private.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 259200 300
api.cam-src-28960.qe.gcp.devcluster.openshift.com.      A     60     10.0.0.2
api-int.cam-src-28960.qe.gcp.devcluster.openshift.com.  A     60     10.0.0.2
*.apps.cam-src-28960.qe.gcp.devcluster.openshift.com.   A     30     162.222.182.233
$ 
$ gcloud dns record-sets list --zone cam-src-28960-pzq7z-private-zone
NAME                                                    TYPE  TTL    DATA
cam-src-28960.qe.gcp.devcluster.openshift.com.          NS    21600  ns-gcp-private.googledomains.com.
cam-src-28960.qe.gcp.devcluster.openshift.com.          SOA   21600  ns-gcp-private.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 259200 300
api.cam-src-28960.qe.gcp.devcluster.openshift.com.      A     60     10.0.0.2
api-int.cam-src-28960.qe.gcp.devcluster.openshift.com.  A     60     10.0.0.2
*.apps.cam-src-28960.qe.gcp.devcluster.openshift.com.   A     30     162.222.182.233
$

Reply:
Here mainly for not changing the original logic, though we are not use how it works in dev's workflow.